src/compositions/templates/case-study/Success/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Main, Sub, Title } from './styled';
+
+// TODO success.sub should be flexible for new lines - edit the fake data in data/case-studies/trainline.yml
```

Comment:
Did you think about how we could handle newlines? I think having `sub` as a markdown block could work and then render it using https://github.com/rexxars/react-markdown

Another option is to have a list of text items instead of a single text item, and render them as separate paragraphs in the component

Reply:
Nuno Campos told me today that css has a feature that handles that. 
 ```
   white-space: pre-line;
```
I don't know if this is the best way but it works! So if data has new lines css will show them!